tests/utf8-bom.txt

```diff
@@ -0,0 +1,14 @@
+Byte Order Marks
+================
+
+Files with byte order marks are properly handled
+
+
+    >>> from findimports import find_imports
+
+    >>> with open('utf8bom.py', 'w') as f: _ = f.write('''\ufeff
```

Comment:
```suggestion
    >>> with open('utf8bom.py', 'w', encoding='UTF-8') as f: _ = f.write('''\ufeff
```

otherwise this test fails on non-UTF-8 locales:

```
$ TOX_TESTENV_PASSENV=LC_ALL LC_ALL=en_US tox -e py36
GLOB sdist-make: /home/mg/src/findimports/setup.py
py36 inst-nodeps: /home/mg/src/findimports/.tox/.tmp/package/1/findimports-1.5.3.dev0.zip
py36 installed: findimports @ file:///home/mg/src/findimports/.tox/.tmp/package/1/findimports-1.5.3.dev0.zip
py36 run-test-pre: PYTHONHASHSEED='2109217123'
py36 run-test: commands[0] | python testsuite.py
.............................F.
======================================================================
FAIL: tests/utf8-bom.txt
Doctest: utf8-bom.txt
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mg/opt/python36/lib/python3.6/doctest.py", line 2199, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for utf8-bom.txt
  File "tests/utf8-bom.txt", line 0

----------------------------------------------------------------------
File "tests/utf8-bom.txt", line 9, in utf8-bom.txt
Failed example:
    with open('utf8bom.py', 'w') as f: _ = f.write('''\ufeff
    import sys
    ''')
Exception raised:
    Traceback (most recent call last):
      File "/home/mg/opt/python36/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest utf8-bom.txt[1]>", line 3, in <module>
        ''')
    UnicodeEncodeError: 'latin-1' codec can't encode character '\ufeff' in position 0: ordinal not in range(256)
----------------------------------------------------------------------
File "tests/utf8-bom.txt", line 12, in utf8-bom.txt
Failed example:
    for imp in find_imports('utf8bom.py'):
        print(imp)
Differences (ndiff with -expected +actual):
    - ImportInfo('sys', 'utf8bom.py', 2, None)


----------------------------------------------------------------------
Ran 31 tests in 0.049s

FAILED (failures=1)
ERROR: InvocationError for command /home/mg/src/findimports/.tox/py36/bin/python testsuite.py (exited with code 1)
________________________________________________ summary ________________________________________________
ERROR:   py36: commands failed
```
```